### PR TITLE
Run stylish-haskell in all files! 💅🏼

### PR DESCRIPTION
**Pull request by kutyel**
The moment has arrived!

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
```
Comment (serras) [inline .editorconfig]: Why do we need this file?

File: .stylish-haskell.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+steps:
```
Comment (serras) [inline .stylish-haskell.yaml]: Why is this file so big? I thought we were just using the defaults except for the `language_prefix`.

File: .stylish-haskell.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      remove_redundant: true
+      language_prefix: language
+
+  # Remove trailing whitespace
+  - trailing_whitespace: {}
+
+columns: 100
+newline: native
+cabal: true
+language_extensions:
```
Comment (serras) [inline .stylish-haskell.yaml]: Why is there a list of language extensions here?

File: stack-nightly.yaml [hunk 4]
```diff
@@ -21,3 +21,5 @@ extra-deps:
 - http2-client-grpc-0.8.0.0
 - avro-0.4.5.4
 - language-protobuf-1.0
+- github: jaspervdj/stylish-haskell
```
Comment (serras) [inline stack-nightly.yaml]: These should be removed. Instead, please write a `development.md` file in which you explain that we are using an unreleased branch of `stylish-haskell`.

File: stack.yaml [hunk 5]
```diff
@@ -20,6 +20,8 @@ extra-deps:
 - http2-client-grpc-0.8.0.0
 - avro-0.4.5.4
 - language-protobuf-1.0
+- github: jaspervdj/stylish-haskell
```
Comment (serras) [inline stack.yaml]: To be removed

File: .editorconfig (hunk 1 above)
Comment (kutyel) [reply .editorconfig]: https://editorconfig.org/ 😄

File: .stylish-haskell.yaml (hunk 2 above)
Comment (kutyel) [reply .stylish-haskell.yaml]: Apparently you need to provide the whole YAML file from which stylish-haskell needs to read 😞

File: .stylish-haskell.yaml (hunk 3 above)
Comment (kutyel) [reply .stylish-haskell.yaml]: This is for the parser, AFAIK

**Pull request merged**